### PR TITLE
The first JVM-only extension

### LINKED ISSUE #894: Elastichsearch Rest support

### PR DESCRIPTION
**Pull request by ppalaga**
Fix #894

### CODE REVIEW

**Comment by ppalaga** [comment]
Any feedback is appreciated.

File: tooling/create-extension-templates/runtime-pom.xml [hunk 1]
```diff
@@ -30,9 +30,10 @@
 
     <artifactId>[=artifactId]</artifactId>
     <name>[=namePrefix][=nameBase][=nameSegmentDelimiter]Runtime</name>
+    <description>[=description]</description>
 
     <properties>
-        <firstVersion>1.0.0-M6</firstVersion>
+        <firstVersion>1.0.0-M5</firstVersion>
```
Comment (jamesnetherton) [inline tooling/create-extension-templates/runtime-pom.xml]: Shouldn't it be M6?

File: tooling/create-extension-templates/quarkus-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - "[=keyword]"
+[/#list]
+[/#if]
+  guide: "[=guideUrl]"
+[#if categories?size > 0 ]  categories:
+[#list categories as category]
+  - "[=category]"
+[/#list]
+[/#if]
+  status: "[=nativeSupported?then('stable', 'preview')]"
```
Comment (jamesnetherton) [inline tooling/create-extension-templates/quarkus-extension.yaml]: Auto generated `quarkus-extension.yaml` :heart_eyes:

File: tooling/create-extension-templates/runtime-pom.xml (hunk 1 above)
Comment (ppalaga) [reply tooling/create-extension-templates/runtime-pom.xml]: +1

**Comment by ppalaga** [comment]
There are two things missing:

* Running the JVM-only tests by the CI
* Listing the JVM-only extensions in the docs

**Comment by ppalaga** [comment]
> There are two things missing:
> 
> * Running the JVM-only tests by the CI

It should already be covered by the `build` jobs's `mvn clean install`.

> * Listing the JVM-only extensions in the docs

Can be done as a followup.

**Comment by ppalaga** [comment]
ec0ec5f:

* Fixed the M5/6 issue found by @jamesnetherton 
* Replaced the plugin in the docs
* Used the proper release of the plugin to generate the extension

**Comment by ppalaga** [comment]
Another followup: some source checks may need a reconfig

**Comment by ppalaga** [comment]
@jamesnetherton reviewed and agreed to merge

**Pull request merged**